docker/docker.go

```diff
@@ -69,6 +69,32 @@ func DoRollingUpdateContainer(dClient *client.Client, imageCfg *container.Config
 	return err
 }
 
+func DoRemoveContainer(dClient *client.Client, containerName, hostname string) error {
+	logrus.Infof("[down/%s] Checking if container is running on host [%s]", containerName, hostname)
+	// not using the wrapper to check if the error is a NotFound error
+	_, err := dClient.ContainerInspect(context.Background(), containerName)
```

Comment:
Why not use IsContainerRunning() ? or at least  InspectContainer() ?

Reply:
This is because i need to inspect the error to make sure its not a notfound error